app/views/donors/admin_index.html.erb

```diff
@@ -212,7 +209,6 @@
                                     </thead>
                                     <tbody ng-init="adminFoodPickups()">
                                         <tr class="odd gradeX" ng-repeat='pickup in approvedPickups'>
-                                            <td>{{pickup.picture}}</td>
```

Comment:
Did we want to keep the pictures in the tables? I thought we agreed not to? Please let me know if that's changed and I will put it back.